### PR TITLE
Fix paddle_math.test_paddle_conj #19276

### LINKED ISSUE #19276: Fix paddle_math.test_paddle_conj
|   |   |
|---|---|
|numpy|<a href="https://github.com/unifyai/ivy/actions/runs/6197499538/job/16826154279"></a>
|jax|<a href="https://github.com/unifyai/ivy/actions/runs/6197499538/job/16826154279"></a>
|tensorflow|<a href="https://github.com/unifyai/ivy/actions/runs/6197499538/job/16826154279"></a>
|torch|<a href="https://github.com/unifyai/ivy/actions/runs/6197499538/job/16826154279"></a>
|paddle|<a href="https://github.com/unifyai/ivy/actions/runs/6197499538/job/16826154279"></a>

### PR DESCRIPTION
**Pull request by so-dipe**
Close #19276

### CODE REVIEW

File: ivy/functional/backends/numpy/experimental/elementwise.py [hunk 1]
```diff
@@ -268,7 +268,7 @@ def conj(
     out: Optional[np.ndarray] = None,
 ) -> np.ndarray:
     ret = np.conj(x, out=out)
-    if x.dtype == np.bool:
+    if x.dtype == bool:
```
Comment (hello-fri-end) [inline ivy/functional/backends/numpy/experimental/elementwise.py:271]: Could you please explain why you changed this and how this fixes the test?

**Review by hello-fri-end — COMMENTED** [review]
Hey @so-dipe ! Thanks for looking into this failing this. Would you please address the comment I've left?

File: ivy/functional/backends/numpy/experimental/elementwise.py (hunk 1 above)
Comment (so-dipe) [reply ivy/functional/backends/numpy/experimental/elementwise.py:271]: numpy deprecated the np.bool and replaced it with python built in bool. that is why the test was failing.

**Comment by so-dipe** [comment]
@hello-fri-end numpy deprecated the np.bool and replaced it with python built in bool. that is why the test was failing. So all I had to do was replace it in the code too.

File: ivy/functional/backends/numpy/experimental/elementwise.py (hunk 1 above)
Comment (hello-fri-end) [reply ivy/functional/backends/numpy/experimental/elementwise.py:271]: Thanks for the explanation!

**Review by hello-fri-end — APPROVED** [review]
Hey @so-dipe! Thanks for the explanation and I'm sorry for the late reply here. Merging this now.

**Pull request merged**